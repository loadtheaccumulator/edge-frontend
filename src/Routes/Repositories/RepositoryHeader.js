import React from 'react';
import PageHeader, {
  PageHeaderTitle,
} from '@redhat-cloud-services/frontend-components/PageHeader';

const RepositoryHeader = () => {
  return (
    <PageHeader>
      <>
        <PageHeaderTitle title="Custom repositories" />
        <p className="pf-u-mt-sm">
          Add custom repositories to build RHEL for Edge images with additional
          packages.
        </p>
      </>
    </PageHeader>
  );
};

export default RepositoryHeader;
