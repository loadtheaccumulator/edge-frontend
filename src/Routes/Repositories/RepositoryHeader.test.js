import React from 'react';
import { render, screen } from '@testing-library/react';
import RepositoryHeader from './RepositoryHeader';

describe('RepositoryHeader', () => {
  it('renders correctly', async () => {
    const { container } = render(<RepositoryHeader />);

    expect(screen.getByRole('heading', 'Custom repositories')).toBeDefined();
    expect(
      screen.getByText(
        'Add custom repositories to build RHEL for Edge images with additional packages.'
      )
    ).toBeDefined();
    expect(container.querySelector('div')).toMatchSnapshot();
  });
});
