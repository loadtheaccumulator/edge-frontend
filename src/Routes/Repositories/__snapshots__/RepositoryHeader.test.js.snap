// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RepositoryHeader renders correctly 1`] = `null`;
